std_msgs/Header header
geometry_msgs/Pose2D pose
uint8 not_change=100
bool end
